### DISCUSSION TITLE
Use images from .next/static folder during runtime

### DISCUSSION THREAD

**Discussion by otkrivashkin** [vercel/next.js#36709] [Help]
Hello everyone!
Is it a good idea to use images from .next/static folder during runtime?
During the build script, I will put images  in that folder and use them. The idea to improve the performance.

**Comment by ephraimduncan** [comment]
I pretty much think it is not a good idea to do this. First of all, the `.next` folder is temporary. When the application is deployed, a new one is created different that is why it is added to your `.gitignore` to prevent you from pushing it. Also, the `public` folder in a Next.js main application is really optimized to serve static files. Files inside public can then be referenced by your code starting from the base URL (/). You also get asset bundling and compression benefits when you place your static files in the `public` folder.
